package.json

```diff
@@ -184,11 +182,13 @@
     "lodash.set": "^4.3.2",
     "markdown-table": "^3.0.2",
     "mkdirp": "^2.1.5",
+    "node-fetch": "^2.6.7",
```

Comment:
maybe node-fetch can be removed entirely as cross-fetch is also used.

Reply:
I had that thought too, but I think that might be a bigger scope (so different PR) due to usage being in the dev CDN which is in Node.js so being specific might be useful? Not 100% sure either way